app/src/main/cpp/skyline/audio/resampler.cpp

```diff
@@ -10,137 +13,135 @@ namespace skyline::audio {
         i32 d;
     };
 
```

Comment:
You kinda ruined the formatting here, I think this is a case where manual is best

Reply:
I fixed it somewhat, the problem is that the Android Studio formatter won't allow manual formatting in a lot of cases as it'll revert whatever was done and there isn't a good way to prevent this.

Reply:
Yeah, just copy and paste what was there before. It's fine since we aren't doing many changes to resampler in the future anyway

Reply:
I would but I generally select the entirety of the C++ folder when reformatting, it's bound to get back to this eventually as we can't really add a contributing guideline of "Under no circumstance shall `resampler.cpp` be reformatted".

Reply:
we can just say it when it happens and they can revert the changes the formatter did

Reply:
// @formatter:off
...
// @formatter:on

Reply:
prevents formatting